kit/src/elements/input/mod.rs

```diff
@@ -78,10 +79,16 @@ pub fn validate_no_whitespace(val: &str) -> Option<ValidationError> {
     None
 }
 
-pub fn validate_alphanumeric(val: &str) -> Option<ValidationError> {
-    if !val.chars().all(char::is_alphanumeric) {
+// Default to requireing alpha-numeric inputs, unless ignore_colon override is set on the input field
+pub fn validate_alphanumeric(val: &str, ignore_colon: bool) -> Option<ValidationError> {
+    let mut val_no_colons = val.to_string();
+    if val.contains(":") && ignore_colon {
```

Comment:
it might be better to check the boolean condition `ignore_colon` first. that way, if it is false, no further operations happen. 

maybe the statement could be 
```
if ignore_colon {
    val.retain(|c| c != ':');
}

Reply:
that's cleaner, thank you!